dashing/distribution.yaml

```diff
@@ -1579,6 +1579,12 @@ repositories:
         release: release/dashing/{package}/{version}
       url: https://github.com/ros-gbp/ompl-release.git
       version: 1.4.2-2
+  optris_drivers:
+    doc:
+      type: git
+      url: https://github.com/evocortex/optris_drivers2.git
+      version: master
+    status: maintained
```

Comment:
Just a head's up @w-haag there are not currently doc jobs for packages in the Dashing distribution. Would you like to add a source entry as well?

Reply:
Thank you for the heads-up. As the optris_drivers need an additional library to compile, I'd like to take care of rosdep first and add the source entry once the prerelease tests finish successfully.